src/test/java/core/basesyntax/tests/TestReader.java

```diff
@@ -0,0 +1,86 @@
+package core.basesyntax.tests;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.TransactionDto;
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.impl.CsvFileReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+import org.junit.After;
+import org.junit.Test;
+
+public class TestReader {
+    private static final String FILE_PATH_ONE_LINE =
+            "src/main/resourcestests/fruitShopOneLine.CSV";
```

Comment:
```suggestion
            "src/test/resourcestests/fruitShopOneLine.csv";
```
let's move all files that are used in tests to `src/test/` folder